### ISSUE TITLE
Experiment module:  update status options

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#586]
For the experiment module, please can we make the following changes to each of the Status options:

**Researchers**: no change. The full list should read 'Active' and 'Archived'
**Research Programs**: Please can we add _Completed_ so the whole list reads 'Requested, Active, Completed, Archived'
**Proposals**: Please can we removed _Postponed_ (replaced with _Cancelled_) so the full list reads 'Draft, Submitted, Approved, Rejected, Planning, Completed, Cancelled, Archived'
**Experiments**: Please can we add _Completed_ and _Cancelled_ so the full list reads 'Requested, Planning, Active, Completed, Cancelled, Archived'.
**Designs**: As per Experiments
**Plans**: ** Please can we change _Terminated_ to _Cancelled_ so the full list reads 'Requested, Planning, Active, Completed, Cancelled, Archived' (the same as Experiments and Designs)

In addition, we'd also like to change the status options for Land, Plant and Structure Assets to

**Land and Structure Assets:** Please can we add 'Completed' and 'Discontinued' so the full list reads 'Active, Completed, Discontinued, Archived'
**Plant Assets:** Please can we add 'Harvested' and 'Discontinued' so the full list reads 'Active, Harvested, Discontinued, Archived.'

I am not currently sure how to deal with plot assets, and think this might need some discussion.

A related issue here:
- https://github.com/Rothamsted-Ecoinformatics/farm_rothamsted/issues/585

**Comment by aislinnpearson** [comment]
Possible error to keep an eye out for is instance where changing the status options affect the UI, so only _active_ assets will be displayed. May need to be considered in further UI issues.

**Comment by paul121** [comment]
> In addition, we'd also like to change the status options for Land, Plant and Structure Assets to

Would it be OK if we hold out on this for another issue? I am a little worried about what will happen if these assets are not only active/archived and would like to think on it some.

**Comment by aislinnpearson** [comment]
Agreed that we would move this to milestone 2.20.0 for additional consideration/ to follow the update to FarmOS core version 3.0 (see issue #609)

@paul121 feel free to close this issue if you are happy.
